cli/domains_test.go

```diff
@@ -0,0 +1,728 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package cli_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/absmach/magistrala/cli"
+	"github.com/absmach/magistrala/internal/testsutil"
+	"github.com/absmach/magistrala/pkg/errors"
+	svcerr "github.com/absmach/magistrala/pkg/errors/service"
+	mgsdk "github.com/absmach/magistrala/pkg/sdk/go"
+	sdkmocks "github.com/absmach/magistrala/pkg/sdk/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+const (
+	createCommand     = "create"
+	getCommand        = "get"
+	updateCommand     = "update"
+	deleteCommand     = "delete"
+	usersCommand      = "users"
+	enableCommand     = "enable"
+	disableCommand    = "disable"
+	connectCommand    = "connect"
+	disconnectCommand = "disconnect"
+	connectionsCmd    = "connections"
+	assignCommand     = "assign"
+	unassignCommand   = "unassign"
+	shareCmd          = "share"
+	unshareCmd        = "unshare"
+)
```

Comment:
see this comment: https://github.com/absmach/magistrala/pull/2347#discussion_r1680677164

Reply:
Am waiting for that [PR](https://github.com/absmach/magistrala/pull/2347) to be merged so that I can remove these lines to avoid conflicts.